src/eureka/lib/apphot.py

```diff
@@ -450,8 +457,13 @@ def apphot(meta, image, ctr, photap, skyin, skyout, betahw, targpos,
 
     # SKY
     # make sky annulus mask
-    skyann = np.bitwise_xor(di.disk(iskyout, ictr, isz),
-                            di.disk(iskyin, ictr, isz))
+    if aperture_shape == "hexagon":
+        skyann = np.bitwise_xor(di.hex(iskyout, ictr, isz),
+                                di.hex(iskyin, ictr, isz))
+    else:
+        skyann = np.bitwise_xor(di.disk(iskyout, ictr, isz),
+                                di.disk(iskyin, ictr, isz))
```

Comment:
Rather than adding several if/else statements and duplicating much of the calls, I recommend doing:
```
if aperture_shape == "hexagon":
    aperFn = di.hex
else:
    aperFn = di.disk

# SKY
# make sky annulus mask
skyann = np.bitwise_xor(aperFn(iskyout, ictr, isz),
                            aperFn(iskyin, ictr, isz))
```
etc., replacing all `di.disk` calls with `aperFn` calls.

Implementing the change using this style allows us to add any general shape in the future with very little work

Reply:
Done.